_health-care/_personal_information/_name_and_general_information.html

```diff
@@ -48,11 +48,20 @@
 
     <div class="row">
       <div class="small-3 columns">
+        <label for="veteran_date_of_birth">Date of Birth</label>
+      </div>
+      <div class="small-9 columns">
+        <input type="text" name="veteran[date_of_birth]" />
+      </div>
+    </div>
+
+    <div class="row">
+      <div class="small-3 columns">
         <label for="veteran_gender">Gender</label>
       </div>
       <div class="small-9 columns">
-        <select name="veteran[gender]" ><option value="0">Male</option>
-          <option value="1">Female</option>
+        <select name="veteran[gender]" ><option value="M">Male</option>
+          <option value="F">Female</option>
           <option value="2">NeedOtherOfficialOptions</option></select>
```

Comment:
All the select/radio boxes are pretty much wrong as I was just mocking up the UI when I first did this. There needs to be a pass done to audit all of them so this is just a note to myself.  It made more sense in the rails code that generated it...